data/migrations/02-createTickets.js

```diff
@@ -0,0 +1,35 @@
+// adding note to make changes
+
+exports.up = knex =>
+  knex.schema.createTable('tickets', table => {
+    table.increments();
+
+    // predefined set of options: inQueue/opened/resolved
+    table.string('status', 128).defaultTo('inQueue');
+
+    table.string('title', 256).notNullable();
+
+    table.text('description').notNullable();
+
+    // what the student tried to fix this issue
+    table.text('tried');
+
+    // student who created the ticket
+    table.integer('student_id')
+      .notNullable()
+      .references('users.id')
+      .onUpdate('CASCADE')
+      .onDelete('CASCADE');
+
+    // do we want user or admin deletion to delete the created ticket?
```

Comment:
I would like to maybe think about archiving them instead of deleting them if possible.